.github/workflows/unit-tests.yml

```
@@ -0,0 +1,17 @@
+    unit-tests:
+        name: M2 Unit Tests
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: extdn/github-actions-m2/magento-unit-tests/8.3@master
+              with:
+                composer_name: grin/module
+                module_name: Grin_Module
+                php_version: 8.3
+                magento_version: 2.4.8
+                project_name: magento/project-community-edition 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)